src/js/components/NestedServiceLinks.js

```diff
@@ -46,11 +46,8 @@ class NestedServiceLinks extends React.Component {
       minorLinkAnchorClassName
     );
     let nestedGroups = this.getServicePathParts();
-    let popCount = 1;
+    const popCount = 1;
```

Comment:
This is a confusing variable name, and used only once, so maybe just inline the `1`.

Reply:
agree.